Information/CenterOfMass.FCMacro

```diff
@@ -1,17 +1,22 @@
 # -*- coding: utf-8 -*-
 #
-# You have to select one or multiple solids and then you launch the macro.
-# You'll have a window listing the solids. You can put the density of your
-# material or choose in a huge list of predefined materials.
-# If you know your densities in another unit system, I used FreeCAD convert
-# system to propose differents units.
-# You can also display where is the center of mass.
-# An import button allows you to quickly load saved densities (even if it's
-# not a .csv file from the macro, as soon as there is a column named "Density"
-# it will be loaded.)
+# Usage:
+# 1. select one or multiple solids
+# 2. launch the macro.
+# 3. You'll have a window listing the solids. You can put the density of your
+#    material or choose from a large list of predefined materials. (If you know
+#    your densities in another unit system, I used FreeCAD convert system to
+#    propose different units.)
 #
+# Options:
+$ * You can also display where is the center of mass.
```

Comment:
I see a dollar sign in the diff. What is it?

Reply:
yikes, probably something i introduced